validator/client/src/integration-test/java/tech/pegasys/teku/validator/client/signer/ExternalSignerAltairIntegrationTest.java

```diff
@@ -60,33 +62,6 @@ public Spec getSpec() {
     return TestSpecFactory.createMinimalAltair();
   }
 
-  @Test
-  void shouldSignAltairBlock() throws Exception {
-    final BeaconBlock block = dataStructureUtil.randomBeaconBlock(10);
```

Comment:
what's the reason we're removing this test?

Reply:
ok, it was an actual issue.... I had intended to loop back on this post the integration test changes, I was hoping it'd come in fixed, and it did... but still didnt work, so it's back and now it works.